hazelcast-sql/src/main/java/com/hazelcast/sql/impl/calcite/schema/HazelcastSchemaUtils.java

```diff
@@ -57,38 +55,33 @@ public static HazelcastSchema createCatalog(Schema schema) {
      * objects such as IMap and ReplicatedMap as well as external tables created by Jet. This approach will not work well
      * should we need a relaxed/dynamic object resolution at some point in future.
      *
-     * @param tableResolvers Table resolver to be used to get the list of existing tables.
      * @return Top-level schema.
      */
     @SuppressWarnings({"unchecked", "rawtypes"})
-    public static HazelcastSchema createRootSchema(List<TableResolver> tableResolvers) {
-        // Create tables.
-        Map<String, Map<String, HazelcastTable>> tableMap = new HashMap<>();
+    public static HazelcastSchema createRootSchema(SqlCatalog catalog) {
+        // Create schemas.
+        Map<String, Schema> schemaMap = new HashMap<>();
+
+        for (Map.Entry<String, Map<String, Table>> currentSchemaEntry : catalog.getSchemas().entrySet()) {
+            String schemaName = currentSchemaEntry.getKey();
+
+            Map schemaTables = new HashMap<>();
```

Comment:
We can use `Map<String, org.apache.calcite.schema.Table>` here and remove the `@SuppressWarnings`

Reply:
Fixed.